### PR TITLE
ares and community expansion icons, left align

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Reduced size of expansion icons
- Made uniformed Ares and Community corp icons
- Left align fan-made icons

image

Don't merge yet. Please wait for @kberg to confirm what color for Ares icon. The current red is the same as CE icon. The picture above used the fourth color here.

image

Check Discord dev room for discussion.

### CODE REVIEW

File: src/components/CreateGameForm.ts [hunk 1]
```diff
@@ -507,13 +507,13 @@ export const CreateGameForm = Vue.component("create-game-form", {
                             <div class="create-game-subsection-label">Fan-made</div>
 
                             <input type="checkbox" name="ares" id="ares-checkbox" v-model="aresExtension">
-                            <label for="ares-checkbox">
-                                <div class="create-game-expansion-icon ares-icon"></div>
```
Comment (kberg) [inline src/components/CreateGameForm.ts:511]: Remove ares-icon from ares.less?

**Review by kberg — APPROVED** [review]
Looks really good. Thank you!

File: src/components/CreateGameForm.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts:511]: removed as requested

**Comment by vsrisuknimit** [comment]
image

tagging @kberg for review

**Pull request merged**